src/features/common/types/plantLocation.d.ts

```diff
@@ -1,8 +1,11 @@
-import type { DateString } from './common';
-import type { Links } from './payments';
-import type { Polygon, Point } from 'geojson';
+import { InterventionTypes } from '@planet-sdk/common';
+import { DateString } from './common';
+import { Links } from './payments';
+import { Polygon, Point } from 'geojson';
+import { INTERVENTION_TYPE } from '../../../utils/constants/intervention';
```

Comment:
Type imports should be explicitly specified. The IDE will warn you about this with

```
All imports in the declaration are only used as types. Use `import type`.`
```